analyzer/src/main/kotlin/managers/Bundler.kt

```diff
@@ -348,11 +348,17 @@ data class GemSpec(
                 RemoteArtifact.EMPTY
             }
 
+            val authors = json["authors"]?.textValue()
+                ?.split(',')
+                ?.map {
+                    it.trim()
+                }?.toSortedSet() ?: sortedSetOf()
+
```

Comment:
What do you think about adding a unit test for this function? ...or adjusting an existing test case to cover the authors?

Reply:
I added a test.

Reply:
Thanks!